book/src/bank-forks.md

```diff
@@ -25,35 +26,35 @@ For example:
 
 The following *active forks* are in the forks DAG
 
-* 4,2,1
-* 5,2,1
-* 6,1
-* 7,1
+* {4, 2, 1}
+* {5, 2, 1}
+* {6, 3, 1}
+* {7, 3, 1}
 
-## Merging into root
+## Squashing
 
-A validator votes for a frozen fork.  The *active fork* connecting the fork
-to the root is merged.  If the *active fork* is longer than
-`Forks::ROLLBACK_DEPTH` the oldest two forks are merged.  The oldest fork in
-the *active fork* is the current root, so the second oldest is a direct
-descendant of the root fork.  Once merged, the current root is updated to the
+A validator votes for a frozen fork.  The active fork connecting the fork to
+the root is *squashed*.  If the active fork is longer than
```

Comment:
Is this sentence serving any purpose?: "The active fork connecting the fork to the root is squashed"
ie. is this a different case than described in the very next sentence?